app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/GetSkillFeedbackResponse.kt

```diff
@@ -10,7 +10,7 @@ data class GetSkillFeedbackResponse(
     val accepted: Boolean = false,
     val message: String = "",
     @SerializedName("feedback")
-    var feedbackList: List<Feedback> = ArrayList(),
+    var feedbackList: ArrayList<Feedback> = ArrayList(),
```

Comment:
Why?

Reply:
What I want is that when we delete the feedback, what we should do is that we should remove that item from the list so that the feedback is not shown to the user. It has been removed from the server. To reload and show the fresh response from the server, we need to relaunch the page. So, to avoid that it would be better to just remove that item from the list. And display the rest items.

Reply:
Your response makes no sense. Why change List to ArrayList

Reply:
Because of the presence of List type, it was not allowing operations like remove, removeAt in the feedbackList. SUch operations are possible in ArrayList type only. So, I changed it.

Reply:
Use MutableList